finality-aleph/src/justification/compatibility.rs

```diff
@@ -0,0 +1,44 @@
+use crate::{
+    crypto::{Signature, SignatureV1},
+    justification::AlephJustification,
+};
+use aleph_bft::{PartialMultisignature, SignatureSet};
+use codec::{Decode, DecodeAll, Encode};
+
+/// Old format of justifications, needed for backwards compatibility.
+#[derive(Clone, Encode, Decode, Debug, PartialEq)]
+pub struct AlephJustificationV1 {
+    pub signature: SignatureSet<SignatureV1>,
+}
+
+impl From<AlephJustificationV1> for AlephJustification {
+    fn from(just_v1: AlephJustificationV1) -> AlephJustification {
```

Comment:
nit: just call the argument `justification`, type annotation is self-explanatory?

Reply:
Done